samples/apps/nobuiltins/nobuiltins.cpp

```diff
@@ -131,6 +133,67 @@ namespace nobuiltins
           ccf::endpoints::ExecuteOutsideConsensus::Locally)
         .set_auto_schema<void, ccf::GetCommit::Out>()
         .install();
+
+      auto get_txid = [this](auto& ctx, nlohmann::json&&) {
+        const auto query_string = ctx.rpc_ctx->get_request_query();
+        const auto query_params = nonstd::split(query_string, "&");
+        for (const auto& query_param : query_params)
+        {
+          const auto& [query_key, query_value] =
+            nonstd::split_1(query_param, "=");
+          if (query_key == "seqno")
+          {
+            kv::SeqNo seqno;
+            const auto qv_begin = query_value.data();
+            const auto qv_end = qv_begin + query_value.size();
+            const auto [p, ec] = std::from_chars(qv_begin, qv_end, seqno);
+            if (ec != std::errc() || p != qv_end)
+            {
+              return ccf::make_error(
+                HTTP_STATUS_BAD_REQUEST,
+                ccf::errors::InvalidQueryParameterValue,
+                fmt::format(
+                  "Query parameter '{}' cannot be parsed as a seqno",
+                  query_value));
+            }
+
+            kv::Consensus::View view;
+            const auto result = get_view_for_seqno_v1(seqno, view);
+            if (result == ccf::ApiResult::OK)
+            {
+              ccf::TxID tx_id;
+              tx_id.view = view;
+              tx_id.seqno = seqno;
+              nlohmann::json out = nlohmann::json::object();
```

Comment:
Why not using an out schema for this endpoint too?

Reply:
No real reason, I've added this now.